### ISSUE TITLE
[REGRESSION] v1.22.0 goes interactive after file execution

### ISSUE THREAD

**Issue by pawel-kow** [DENICeG/go-rriclient#19]
v1.18.0 exits after called with `-f` command allowing calling from a bash script.
```
➜ go install github.com/DENICeG/go-rriclient@v1.18.0

rri5.0 on  main [✘?] took 3.2s 
➜ go-rriclient -e MTEST -f "Mocks/Use-case 5: Quarantine -> Success/01 - Create Domain serverHold/out/UC5_20241210_01/01. Create Contact.kv"
Exec query CREATE{}
RESULT: failed
STID: eeed1900-0342-429e-bd3f-fcf995422511
ERROR: 63200080108 This value of "Action" is not permitted here
INFO: 13000000011 Request was processed in test environment - not valid in real world [testing platform]


rri5.0 on  main [✘?] 
➜ 
```
v1.22.0 goes interactive, so the script would hand waiting for an input.
```
rri5.0 on  main [✘?] 
➜ go install github.com/DENICeG/go-rriclient@v1.22.0

rri5.0 on  main [✘?] took 4.6s 
➜ go-rriclient -e MTEST -f "Mocks/Use-case 5: Quarantine -> Success/01 - Create Domain serverHold/out/UC5_20241210_01/01. Create Contact.kv"
RESULT: failed
STID: c9749b5e-f371-4f7e-958a-2ca15bcadc69
ERROR: 63200080108 This value of "Action" is not permitted here
INFO: 13000000011 Request was processed in test environment - not valid in real world [testing platform]
Interactive RRI Command Line
  type 'help' to see a list of available commands
  use tab for auto-completion and arrow keys for history
RRI{DENIC-54321-RRI@rri.test.denic.de:51131}> 
```

**Comment by Nerzal** [comment]
Fixed in v1.23.0

![image](https://github.com/user-attachments/assets/174c78e7-329a-4c3a-a3db-2e89ccfb8933)

**Comment by pawel-kow** [comment]
works, thx.
